kernel/src/platform/tdp.rs

```diff
@@ -200,10 +199,7 @@ impl SvsmPlatform for TdpPlatform {
     fn eoi(&self) {}
```

Comment:
Should we implement this while we're at it?
```suggestion
    fn eoi(&self) {
        x2apic_eoi();
    }
```

Reply:
Yes, that makes sense. I added it.